regulations/templates/regulations/chrome.html

```diff
@@ -138,12 +138,15 @@ <h2 class="toc-type">Search</h2>
         <div class="drawer-content">
             <form action="{% url 'chrome_search' reg_part %}" method="GET" role="search">
                 <div class="search-box">
-                    <input type="text" name="q" value="{{ q }}" id="search-input" /><button type="submit"><i class="cf-icon cf-icon-search"></i><strong class="icon-text">Search</strong></button>
+                    <input type="text" name="q" value="{{ q }}" id="search-input" title="Search term"/>
+                    <button type="submit">
+                        <i class="cf-icon cf-icon-search"></i><strong class="icon-text">Search</strong>
```

Comment:
Before my time - I think it is being used as a non-semantic container for the navigation icons.  The accessibility tool complains about this practice so it will be removed in another 508c iteration.  Repair: Replace your i elements with em or strong.